openedx/adg/lms/webinars/views.py

```diff
@@ -0,0 +1,59 @@
+"""
+All views for webinars app
+"""
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseServerError
+from django.shortcuts import redirect, render
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext as _
+from django.views import View
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.views.generic.detail import SingleObjectMixin
+
+from .helpers import send_webinar_registration_email
+from .models import Webinar, WebinarRegistration
+
+
+def webinar_description_page_view(request, pk):
+    # TODO This is a temp view and will be properly functional in LP-2623
+    return render(request, 'adg/lms/webinar/description_page.html', context={'pk': pk})
+
+
+@method_decorator(ensure_csrf_cookie, name='dispatch')
+class WebinarRegistrationView(LoginRequiredMixin, SingleObjectMixin, View):
+    """
+    A view to register user in a webinar event or cancel already registered user in a webinar.
+    """
+    model = Webinar
+
+    def post(self, request, pk, action):
+        """
+        Register in a webinar or cancel registration
+        Args:
+            request (HttpRequest): HTTP request
+            pk (int): Webinar id, the primary key of model
+            action (str): Actions to perform on event i.e. `register` or `cancel`
+        Returns:
+            HttpResponse: On success redirect to webinar page otherwise through HTTP error code as per nature
+            of error i.e. 404 or 500
```

Comment:
Kindly add newline before `Args` and `Returns` in accordance with [Google docstring guide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) and [edx docstring guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings).